tests/model_fields/tests.py

```diff
@@ -661,6 +689,15 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+            except (DataError, IntegrityError):
+                pass  # backends such as Postgres don't fail silently
+            else:
+                qs = self.model.objects.filter(value__lte=min_value)
```

Comment:
Is there any notable backends to test against in addition? Oracle or SQL Server?